core/storage/voiceover/gae_models.py

```diff
@@ -161,7 +163,7 @@ def create_new(
         entity_version: int,
         language_accent_code: str,
         voiceovers_mapping: Dict[str, Dict[
-            feconf.VoiceoverType.value, Optional[state_domain.VoiceoverDict]]],
+            str, Optional[state_domain.VoiceoverDict]]],
```

Comment:
why this change? If the dict keys are always values of the `feconf.VoiceoverType` enum, then the old type is more specific, right?